src/pages/Stream/index.tsx

```diff
@@ -42,9 +43,11 @@ const Stream: FC = () => {
 	const [currentStream] = useAppStore((store) => store.currentStream);
 	const [maximized] = useAppStore((store) => store.maximized);
 	const [instanceConfig] = useAppStore((store) => store.instanceConfig);
+	const [timeRangeInitialised, setTimeRangeInitialised] = useState(false);
```

Comment:
Actually, It's not initialization here.

Reply:
Changed the state variable nomenclature.